src/lib/components/colors/Sliders.svelte

```diff
@@ -19,7 +19,9 @@
   let spaceObject = $derived(ColorSpace.get(targetSpace));
 
   // Create a throttled value for each channel
-  const throttled = SLIDERS[format].map(() => _.throttle(sliderGradient, 100));
+  const throttled = $derived(
+    SLIDERS[format].map(() => throttle(sliderGradient, 50)),
+  );
```

Comment:
@jamesnw Without making this `$derived`, it wasn't re-calculating these fns on initial page load when the `format` is read from the has. This seems to fix it.

I also bumped the throttle back to `50`, which still seems to have decent performance for me. 🤷

Reply:
@jgerigmeyer I am running into issues with throttling on page load. As it currently is set up, the gradients are calculated once on page load, and then that value will be used for the next 50ms. This means that they are calculated when the color space is set, but not when the gamut is set.

I'm having trouble figuring out how to force the gradient to re-render on gamut change. 

I tried making the whole `let sliders = $derived` function the thing that is debounced, and then flushed the throttles when the gamut changes, but it still doesn't cause the value of sliders to update. Any ideas?

Reply:
@jamesnw Does what I pushed in https://github.com/oddbird/oddcontrast/commit/7e4a540c55aec52858d2c092c9215f30afa16bb5 work for you? It seems like just adding a reference to `$gamut` inside the `$derived` call is enough to force the throttle fns to be re-created when the gamut changes.

I also removed the `maxDeltaE` setting when `gamut === null || channel === 'alpha'` because it seems like it's unnecessary extra computations -- is that okay?

Reply:
Yeah, that works. 

I also separated out the alpha slider gradient into a separate function in aac9bd3, since they ended up sharing very little code.